core/src/com/mygdx/game/assets/GameScreenAssets.java

```diff
@@ -62,9 +61,17 @@ public GameConfigs getGameConfigs() {
     return gameConfigs;
   }
 
-  @Override
-  public void dispose() {
-    assetManager.dispose();
+  public float getLoadingProgress() {
```

Comment:
Jeśli dobrze rozumiem to zwraca procentową wartość załadowych assetów?

Reply:
Tak, ogólnie to jest to trochę średnie, bo ten procent nie jest tylko dla GameScreenAssets, ale też bierze pod uwagę MenuScreenAssets. Mógłbym to zrobić trochę inaczej, czyli zrobić, żeby LoadingScreen brał ten procent bezpośrednio z instacji AssetManagera.